run_local_tests.sh

```diff
@@ -104,7 +104,7 @@ function run_local_tests() (
   if [[ -n "${tests}" ]] ; then
     echo "================================="
     echo "Running qemu_uefi tests"
-    test_run "${arch}" qemu_uefi "${tests}"
+    test_run "${arch}" qemu_uefi ${tests}
```

Comment:
This is just a quick fix; we need a better way to pass options to kola in general. But this needs to be worked into `ci-automation` and is a larger change. Currently we can "officially" only pass lists of tests. This leads to, among other things, kola command line options being dropped on re-runs because they don't pop up as "failed tests".

I'll think about this.